config/rbac/imagejob_pods_role_binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: imagejob-pods-rolebinding
```

Comment:
I would keep the naming consistent with what it is, a `clusterrolebinding` and not just a `rolebinding`. This would apply to the filename too